docs/technical_reference/unit_models/energy_recovery_device.rst

```diff
@@ -0,0 +1,58 @@
+Energy Recovery Device
+======================
+
+.. index::
+   pair: watertap.unit_models.energy_recovery_device;energy_recovery_device
+
+This energy recovery device unit model is a class of pressure changer which is based on the `IDAES pressure changer <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/pressure_changer.html>`_
+and makes the following assumptions:
+
+   * supports a single liquid phase only
+   * supports steady-state only
+   * supports isothermal pump only
+
```

Comment:
I would add the following: 

This energy recovery unit is meant to be representative of a reversible pump or Pelton turbine.

Reply:
Added